bridge/cypress/integration/uniform-integrations.spec.ts

```diff
@@ -425,6 +426,27 @@ describe('Add webhook subscriptions', () => {
   });
 });
 
+describe('Add secret from webhook subscriptions', () => {
```

Comment:
Could you extend the UI tests with the following?
- [ ] `should prevent data loss / navigation if filter was changed`
- [ ] `should prevent data loss / navigation if task was changed`
- [ ] `should prevent data loss / navigation if task suffix was changed`
- [ ] `should prevent data loss / navigation if webhook was changed`
- [ ] `should lose data and navigate to secret creation`

Does this prevent-data-loss change also include subscriptions that are not for the webhook service?

Reply:
The onChangeGuard also works for non webhook-service subscriptions as the 'Create subscription
' form is the same.